chart/kubeapps/templates/_helpers.tpl

```diff
@@ -6,6 +6,13 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{- define "kubeapps.parseAPIServiceURL" -}}
+{{- $parsed := urlParse . }}
+{{- $baseURL := (printf "%s://%s" (get $parsed "scheme") (get $parsed "host")) }}
+{{- $path := get $parsed "path" }}
+{{ dict "baseURL" $baseURL "path" $path | toYaml }}
```

Comment:
@absoludity this template "returns" a YAML string which can be composed (in the caller) with `| fromYaml` so that I can "return" a dict... 🤯